v1/contracts/UnsignedFixedPoint.sol

```diff
@@ -0,0 +1,44 @@
+pragma solidity ^0.5.0;
+
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+
+/**
+ * @title Library for fixed point arithmetic on uints
+ */
+library UnsignedFixedPoint {
+
+    using SafeMath for uint;
+
+    // Supports 18 decimals. E.g., 1e18 represents "1", 5e17 represents "0.5".
+    uint private constant FP_SCALING_FACTOR = 10**18;
+
+    struct FixedPointUint {
+        uint value;
+    }
+
+    function add(FixedPointUint memory a, FixedPointUint memory b) internal pure returns (FixedPointUint memory) {
```

Comment:
nit: docstrings or do we only do those for public/external functions?

Reply:
They're worth adding IMO, added them. Though I think the docgen tool ignores internal functions.